components/motor/fake/motor.go

```diff
@@ -57,23 +67,8 @@ func (cfg *Config) Validate(path string) ([]string, error) {
 }
 
 func init() {
-	resource.RegisterComponent(motor.API, model, resource.Registration[motor.Motor, *Config]{
-		Constructor: func(
-			ctx context.Context,
-			deps resource.Dependencies,
-			conf resource.Config,
-			logger logging.Logger,
-		) (motor.Motor, error) {
-			m := &Motor{
-				Named:  conf.ResourceName().AsNamed(),
-				Logger: logger,
-				OpMgr:  operation.NewSingleOperationManager(),
-			}
-			if err := m.Reconfigure(ctx, deps, conf); err != nil {
-				return nil, err
-			}
-			return m, nil
-		},
+	resource.RegisterComponent(motor.API, motorModel, resource.Registration[motor.Motor, *Config]{
+		Constructor: NewMotor,
```

Comment:
Cna you chekc if the other fakes follow this naming scheme?
```suggestion
		Constructor: NewFakeMotor,
```
And change it if so.

Reply:
majority don't have fake in the name